Tests/conf.json

```diff
@@ -2263,11 +2263,6 @@
             "integrations": "Exabeam",
             "playbookID": "Exabeam - Test"
         },
-        {
-            "integrations": "DomainTools Iris",
```

Comment:
can we also remove the integration from `skip`?

Reply:
@yaakovi I don't see that the integration is skipped... where are you looking?